src/main/java/org/jvnet/hudson/test/SupportLogFormatter.java

```diff
@@ -21,26 +21,25 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
 package org.jvnet.hudson.test;
 
 // COPIED from com.cloudbees.jenkins.support @ 2.42-SNAPSHOT
 
-/***********************************************
- * DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
- * IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
- ***********************************************/
+// *********************************************
+// DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
+// IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
+// *********************************************
 import java.io.PrintStream;
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.logging.Formatter;
 import java.util.logging.LogRecord;
-/***********************************************
- * DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
- * IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
- ***********************************************/
+// *********************************************
+// DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
+// IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
+// *********************************************
```

Comment:
Err, what? :) Why this?

Reply:
A Javadoc describes a class, method or field - also see https://stackoverflow.com/a/43874598/2847174.
IntelliJ IDEA marks everything as "dangling".
Therefore I changed this to "normal" comments.